src/main/java/com/turkcell/staj/dtos/packages/requests/RequestUpdatePackageDTO.java

```diff
@@ -0,0 +1,25 @@
+package com.turkcell.staj.dtos.packages.requests;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class RequestUpdatePackageDTO {
+
+    private int packageId;
```

Comment:
id alanlarında negatif ve string değerlerinin validasyonu için şu anno eklenebilir.
`@Min(value = 1, message = "Id must be a positive integer greater than or equal to 1")`